webpack.config.js

```diff
@@ -65,16 +65,18 @@ module.exports = {
     })
   ],
   devServer: {
-    host: 'localhost',
-    port: 8080,
+    host: (process.env.USER === 'root') ? '0.0.0.0' : 'localhost',
```

Comment:
We shouldn't be using the dev server at all in the deployed version, so I don't think these changes are necessary...

Reply:
Got it